tests/integration/ceph_flex_test.go

```diff
@@ -81,10 +81,20 @@ func (s *CephFlexDriverSuite) SetupSuite() {
 	s.namespace = "flex-ns"
 	s.pvcNameRWO = "block-persistent-rwo"
 	s.pvcNameRWX = "block-persistent-rwx"
-	useHelm := false
-	mons := 1
-	rbdMirrorWorkers := 1
-	s.op, s.kh = StartTestCluster(s.T, flexDriverMinimalTestVersion, s.namespace, "bluestore", useHelm, false, "", mons, rbdMirrorWorkers, installer.VersionMaster, installer.OctopusVersion, false)
+
+	flexTestCluster := TestCluster{
+		namespace:        s.namespace,
+		storeType:        "bluestore",
+		storageClassName: "",
+		useHelm:          false,
+		usePVC:           false,
+		mons:             1,
+		rbdMirrorWorkers: 1,
+		rookCephCleanup:  false,
+		withoutOSDs:      true,
```

Comment:
`withoutOSDs` should be `false` for all of the existing test suites. We only want that `false` in the new test suite that would be run daily, right?

Reply:
Yes